src/Providers/FindologicServiceProvider.php

```diff
@@ -59,6 +59,10 @@ public function boot(
             return;
         }
 
+        if (!$searchService->isAliveTest()) {
```

Comment:
Using the implementation as described in the story should work now since the alive test will not be performed in any other place.